runtime/flang/stat_linux.c

```diff
@@ -9,11 +9,13 @@
  * \brief Fill in statistics structure (Linux version)
  */
 
+#ifndef _WIN32
```

Comment:
Since definition part of functions guarded maybe leave a fixme or todo comment here? Should we support it for Windows?

Reply:
will do. yes, we should support it, but mainly focused on getting to basic functionality now.